models/prices/solana/prices_solana_schema.yml

```diff
@@ -22,7 +22,9 @@ models:
           - accepted_values:
               values: [ "solana" ]
       - name: contract_address
-        description: "Contract address of the token, if any"
+        description: "Contract address in varbinary typed of the token, if any"
+      - name: raw_contract_address
+        description: "Raw contract address in varchar of the token, if any"
```

Comment:
I am thinking if we can add a new column `raw_contract_address` for non-evm chains' price tables, to keep their original contract address in varchar typed.

Reply:
the original one has to be varbinary otherwise this will break all queries that join on it. I don't think we should do this change now. It's a bit unfortunate that we have to use from_base58 like this but if we change this it would have to be part of a bigger decision.

Reply:
yeah, i'll revert those changes for now. it was known at the time of implementation of solana addresses into prices that the data type would be converted. we can revisit in future PR if users want it reworked.